v20.1/build-a-spring-app-with-cockroachdb-jdbc.md

```diff
@@ -0,0 +1,834 @@
+
+For more details about the application code, see [Implementation details](#implementation-details).
+
+### Query the database
+
+#### Reads
+
+The `http://localhost:8080/account` endpoint returns information about all accounts in the database. `GET` requests to these endpoints are executed on the database as simple `SELECT` statements.
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ curl -X GET http://localhost:8080/account | json_pp
+~~~
+
+~~~
+{
+   "_embedded" : {
+      "accounts" : [
+         {
+            "_links" : {
+               "self" : {
+                  "href" : "http://localhost:8080/account/1"
+               }
+            },
+            "balance" : 500,
+            "name" : "Alice",
+            "type" : "asset"
+         },
+         {
+            "_links" : {
+               "self" : {
+                  "href" : "http://localhost:8080/account/2"
+               }
+            },
+            "balance" : 500,
+            "name" : "Bob",
+            "type" : "expense"
+         },
+         {
+            "_links" : {
+               "self" : {
+                  "href" : "http://localhost:8080/account/3"
+               }
+            },
+            "balance" : 500,
+            "name" : "Bobby Tables",
+            "type" : "asset"
+         },
+         {
+            "_links" : {
+               "self" : {
+                  "href" : "http://localhost:8080/account/4"
+               }
+            },
+            "balance" : 500,
+            "name" : "Doris",
+            "type" : "expense"
+         }
+      ]
+   },
+   "_links" : {
+      "self" : {
+         "href" : "http://localhost:8080/account?page=0&size=5"
+      }
+   },
+   "page" : {
+      "number" : 0,
+      "size" : 5,
+      "totalElements" : 4,
+      "totalPages" : 1
+   }
+}
+~~~
+
+For a single account, specify the account number in the endpoint. For example, to see information about the accounts `1` and `2`:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ curl -X GET http://localhost:8080/account/1 | json_pp
+~~~
+
+~~~
+{
+   "_links" : {
+      "self" : {
+         "href" : "http://localhost:8080/account/1"
+      }
+   },
+   "balance" : 500,
+   "name" : "Alice",
+   "type" : "asset"
+}
+~~~
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ curl -X GET http://localhost:8080/account/2 | json_pp
+~~~
+
+~~~
+{
+   "_links" : {
+      "self" : {
+         "href" : "http://localhost:8080/account/2"
+      }
+   },
+   "balance" : 500,
+   "name" : "Bob",
+   "type" : "expense"
+}
+~~~
+
+The `http://localhost:8080/transfer` endpoint performs transfers between accounts. `POST` requests to this endpoint are executed as writes (i.e., `INSERTS` and `UPDATES`) to the database.
```

Comment:
Could link to `INSERT` and `UPDATE`.  Plurals should be formatted as `INSERT`s, I think.